### ISSUE TITLE
Configuration for page-toc .markdown fails

### ISSUE THREAD

**Issue by moxiegirl** [aleung/gitbook-plugin-page-toc#4]
```
"selector": ".markdown-section h2"
```

I was hoping to only show the h2 items in the toc. So I used this configuration but it fails. It keeps adding the `<h1>` values.

**Comment by dawncold** [comment]
@moxiegirl this works for me.
```json
{
  "plugins": [ "page-toc" ],
  "pluginsConfig": {
    "page-toc": {
      "selector": ".markdown-section h2",
      "position": "top",
      "showByDefault": true
    }
  }
}
```
![screen shot 2018-06-22 at 5 19 19 pm](https://user-images.githubusercontent.com/502856/41769000-7134a56e-7640-11e8-8fa7-3e1ba738400b.png)

**Comment by samlinpaf** [comment]
@dawncold Yep, that works. Unfortunately the `"page-doc"` element isn't specified on this page, which is the top result on Google: https://plugins.gitbook.com/plugin/page-toc

**Comment by moxiegirl** [comment]
@dawncold I'll give that try, thank you.

**Comment by aleung** [comment]
The readme was fixed by #7. Need to publish a new version to get https://plugins.gitbook.com/plugin/page-toc updated.

**Comment by aleung** [comment]
Published v1.1.1. Leave comment if still has issue.
